src/Team-Services-Bot.Api/Web.config

```diff
@@ -14,7 +14,7 @@
   http://go.microsoft.com/fwlink/?LinkId=301879
   -->
 <configuration>
-  <appSettings configSource="AppSettings.config" />
+  <appSettings configSource="AppSettings.config.user" />
```

Comment:
That is intentional. I strongly believe that hacking of source control is really dangerous. Having AppAettings.config.user allows you to have your local settings for debugging and that file wouldn't go to source control.